docs/pages/management/guides/awsoidc-integration.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: AWS OIDC Integration
+description: How to connect your AWS account with Teleport and provide access to AWS resources.
+---
+
+This guide explains how to set up the Teleport AWS OIDC integration.
+
+With the AWS OIDC integration you will no longer need to deploy Teleport agents in AWS manually for most use cases.
+The following features use an AWS OIDC integration to interact with AWS:
+- [External Audit Storage](../../choose-an-edition/teleport-cloud/external-audit-storage.mdx)
+- RDS Auto Discovery
+- EC2 Auto Discovery
+- [Access Graph AWS Sync](../../access-controls/access-graph/aws-sync.mdx)
+
+It targets users who would prefer a more manual approach or which to use a more Infrastrcutre as Code oriented flow.
+
+As an alternative to this guide, you can use the Teleport Web UI (Access Management / Enroll New Integration).
+
+## How it works
+Teleport is added as an Identity Provider in AWS by exposing the `openid-configuration` and public keys in a public S3 bucket.
```

Comment:
```suggestion
Teleport is added as an [OpenID Connect identity provider](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc.html) to establish trust with your AWS account and assume a configured IAM role in order to access AWS resources. 

For this to work, the `openid-configuration` and public keys are exposed in a public S3 bucket. The integration requires no extra configuration or services to run.
```

Added a link, fixed capitalization, and indicated at a high level what an AWS OIDC integration does. Otherwise, this seems to launch into specifics too early.